pages/pipelines/build_exports.md

```diff
@@ -83,10 +83,37 @@ To configure build exports for your organization, you'll need to prepare an S3 b
 
 Your Buildkite Organization ID (UUID) can be found on the settings page described in the next section.
 
+### Prepare your Google Cloud Storage bucket
+
+* Read and understand [Google Cloud Storage security best practices](https://cloud.google.com/security/best-practices) and [Best practices for Cloud Storage](https://cloud.google.com/storage/docs/).
+* Your bucket must have a policy allowing our Buildkite service-account access as described here and demonstrated in the example below¹.
+  - Assign Buildkite's service-account `buildkite-production-aws@buildkite-pipelines.iam.gserviceaccount.com` the `"Storage Object Creator"`.
+  - Scope the `"Storage Object Creator"` role using IAM Conditions to limit access to objects matching the prefix `buildkite/build-exports/org=YOUR-BUILDKITE-ORGANIZATION-UUID/*`¹.
+* Your bucket must grant our Buildkite service-account (`buildkite-production-aws@buildkite-pipelines.iam.gserviceaccount.com`) `storage.objects.create` permission.
+* Your bucket should use modern Google Cloud Storage security features and configurations, for example (but not limited to):
+  - [Public access prevention](https://cloud.google.com/storage/docs/public-access-prevention) to prevent accidental misconfiguration leading to data exposure.
+  - [Access control lists](https://cloud.google.com/storage/docs/access-control/lists) to ensure your GCP (Google Cloud Provider) account owns the objects written by Buildkite.
+  - [Data encryption options](https://cloud.google.com/storage/docs/encryption).
+  - [Object versioning](https://cloud.google.com/storage/docs/object-versioning) to help recover objects from accidental deletion or overwrite.
+* You may want to use [GCS Object Lifecycle Management](https://cloud.google.com/storage/docs/lifecycle) to manage storage class and object expiry.
+
+¹ Your IAM Conditions should look like this, with `YOUR-BUCKET-NAME-HERE` and `YOUR-BUILDKITE-ORGANIZATION-UUID` substituted with your details:
+
```

Comment:
TODO: Did we want to add anything using gsutil?

Reply:
cc: @pda @mitchbne

Reply:
@123sarahj123 I don't think so, it looks like the Google Cloud Platform [Storage Transfer Service](https://cloud.google.com/storage-transfer-service) captures everything that we were considering when we floated to `gsutil` tool internally